Cython/Compiler/Nodes.py

```diff
@@ -9079,7 +9079,7 @@ class ParallelStatNode(StatNode, ParallelNode):
                                 construct (replaced by its compile time value)
     """
 
-    child_attrs = ['body', 'num_threads']
+    child_attrs = ['body', 'num_threads', 'if_']
```

Comment:
As I have mentioned, `if_` is not final version. I wanted to have something just to finish implementation. I am open to any suggestion. I like `use_threads_if` more, so I will use it instead of `if_`. If there will be consensus on other name I am fine to change it again.